server.js

```diff
@@ -20,13 +15,95 @@ mongoose.Promise = Promise;
 const port = process.env.PORT || 8080;
 const app = express();
 
+//Seeding with database
+
+const Song = mongoose.model("Song", {
+  id: Number,
```

Comment:
there's no need to pass an id to your schema, remove `id` as Mongoose automatically assigns a unique `_id` field to each document, so you don't need to define an `id` field :) (also from the data set)